aws_lambda_powertools/event_handler/openapi/encoders.py

```diff
@@ -14,6 +14,7 @@
 from pydantic.types import SecretBytes, SecretStr
 
 from aws_lambda_powertools.event_handler.openapi.compat import _model_dump
+from aws_lambda_powertools.event_handler.openapi.exceptions import EncoderError
```

Comment:
This might be better as `SerializationError` or `SerDeError` if we ever plan to decode too

Reply:
Great catch @sthulb! I've change the name to `SerializationError`